src/main/resources/static/js/src/HistoryController.js

```diff
@@ -0,0 +1,36 @@
+const app = angular.module("myApp", []);
+app.controller("HistoryController", function($scope,$http) {
+    let myPhone = document.getElementById("from").textContent;
+
+    $http({
+        method: 'GET',
+        url: '/rest/getCallsList'+myPhone
+    }).then(function successCallback(response) {
+        $scope.calls = response.data;
+
+        $scope.styleFunction = function (to, status) {
+            if(status !== 'no-answer') {
+                let incoming = myPhone === to;
+
+                if (incoming) {
+                    return 'incoming';
+                } else {
+                    return 'outgoing';
+                }
+            }else {
+                return 'no-answer';
```

Comment:
isInbound(to) {
return myPhone === to;
 }     
styleFunction(to, status) {
            if (status == "no-answer") {
                return status;
            } else if (isInbound(to)) {
                return "Inbound";
            } else {
                return "Outbound";
            }
        }